content/docs/command-reference/plots/modify.md

```diff
@@ -134,8 +134,8 @@ plots:
 
 ## Example: Template change
 
-_dvc run --plots file.csv ..._ command assign the default template that needs to
-be changed in many cases. A simple command changes the template:
+_dvc stage add --plots file.csv ..._ command assign the default template that
```

Comment:
This should render correctly now:
```suggestion
`dvc stage add --plots file.csv ...` assign the default template that
```
It's just a bit unclear why we're referring to such a specific `stage add` command at the opening of this example...